src/open_inwoner/openklant/constants.py

```diff
@@ -17,3 +19,9 @@ def safe_label(cls, value, default=""):
             if default:
                 return default
             return str(value).replace("_", " ").title()
+
+
+@dataclass
+class KlantenServiceType:
+    esuite = "esuite"
+    openklant2 = "openklant2"
```

Comment:
Seems like an `enum` to me:

```suggestion

class KlantenServiceType(enum.Enum):
    ESUITE = "esuite"
    OPENKLANT2 = "openklant2"
```